data/src/main/scala/cats/data/Or.scala

```diff
@@ -381,6 +374,10 @@ sealed abstract class OrInstances extends OrInstances0 {
       def empty =
         ROr(Monoid[B].empty)
     }
+
+  implicit def OrdShow[A: Show, B: Show]: Show[A Or B] =
```

Comment:
I think you meant `OrShow` not `OrdShow`.

Also `oneAndShow` is lowercase but `OrdShow` is upper case. I don't particularly care which convention cats adopts, but I think it should stick to one.